src/server_test.py

```diff
@@ -55,3 +55,13 @@ def test_fetch_linkedin_redirect_url(self):
         self.assertEqual(server.find_redirect_url("https://www.linkedin.com/redir/redirect"), None)
         self.assertEqual(server.find_redirect_url("https://www.linkedin.com/redir/redirect?url=abc"), None)
         self.assertEqual(server.find_redirect_url("https://www.linkedin.com/redir/redirect?url=https%3A%2F%2Fab.cd"), "https://ab.cd")
+
+    def test_cleanup_extra_ids(self):
+        url = 'https://ab.cd/path1/path2?q=1'
+        config = Config()
+        config.cleanup_extra_ids = ['id1']
+        article = Article(url, config=config)
+        article.set_html("<html><body><p id='id1'>To be cleaned</p><p>This is example paragraph with some text123.</p></body></html>")
```

Comment:
can we instead test for both "artdeco-global-alert-container" and "artdeco-global-alerts-cls-offset" instead
of id1